src/components/AddressHistory.js

```diff
@@ -168,9 +167,8 @@ class AddressHistory extends React.Component {
             <td className="d-none d-lg-table-cell pr-3">{hathorLib.helpers.getTxType(tx)}</td>
             <td className="d-none d-lg-table-cell pr-3">{hathorLib.helpers.getShortHash(tx.tx_id)}</td>
             <td className="d-none d-lg-table-cell pr-3">{dateFormatter.parseTimestamp(tx.timestamp)}</td>
-            <td className={tx.is_voided ? 'voided state' : 'state'}>{statusElement}</td>
-            <td>{tx.is_voided && <span className="voided-element">Voided</span>}</td>
```

Comment:
We don't have the voided information anymore?

Reply:
We don't fecth voided transactions with the new API.

Reply:
Hmmm, was this a design decision? My guess is that we should display voided transactions

It would definitely be possible with the wallet-service backend, we have voided transactions on the `address_tx_history` table

Reply:
It was decided with Trond that we won't show voided transactions anymore in this screen